components/payment-term.jsx

```diff
@@ -82,11 +77,17 @@ export function PaymentTerm({
 				</React.Fragment>
 			),
 			monthlyPrice: () => {},
-			renewsText: () => (
-				<p className="ncf__payment-term__renews-text">
-					Renews monthly unless cancelled
-				</p>
-			),
+			renewsText: (isFixedTermOffer) => {
+				return isFixedTermOffer ? (
+					<p className="ncf__payment-term__renews-text">
+						This subscription is charged monthly and can be cancelled at anytime
+					</p>
+				) : (
+					<p className="ncf__payment-term__renews-text">
+						Renews monthly unless cancelled
+					</p>
+				);
+			},
```

Comment:
Nitpick: Duplicated `<p>` block may be avoiding with something like this.

```suggestion
			renewsText: (isFixedTermOffer) => {
				const textToDisplay = isFixedTermOffer ? 'This subscription is charged monthly and can be cancelled at anytime' : 'Renews monthly unless cancelled';
				return (
					<p className="ncf__payment-term__renews-text">{textToDisplay}</p>
				);
			},
```